### ISSUE TITLE
Ranger failed to properly paste .jpg files and corrupted them.

### ISSUE THREAD

**Issue by alexaldev** [ranger/ranger#1967]
- Operating system and version: Ubuntu 16.04.1
- Terminal emulator and version: urxvt 9.21
- Python version: 2.7.12
- Ranger version/commit: 1.7.1 
- Locale: US

#### Current Behavior
I `cut and deleted` multiple directories from the hard drive which hosts the operation system to an internal drive which I use as backup and 1 directory of them resulted in corrupt files, so now I am not able to access them. They were .jpg files in a single directory and all of them are now corrupted in the internal drive. I am able to see their metadata but not able to load them.  After this operation, ranger fails to load with an exit message described below. 

#### Expected Behavior
I would like to have access to the corrupted image files in the directory which somehow failed to be pasted correctly in the internal drive.

#### Context

I tried to keep a backup from the drive I use for the operation system to an internal drive using ranger and it failed to safely copy a whole directory with files resulting in corrupted files. I do not know how to get them back.
#### Possible Solutions
No idea.
#### Steps to reproduce
1. Create a directory with 4 subdirectories each containing ~100 .jpg files. 
2. Mount an internal drive in `/mnt/[whatever]` directory.
3. Using ranger, use the command `dd`(cut+paste) on the 4 directories to paste them to the internal drive. 
4. Close ranger.
5. Try to open ranger again from the same session. If it crashes, you may have managed to properly reproduce the bug.

#### Traceback

```
Locale: en_US.UTF-8
Current file: None
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/ranger/core/main.py", line 121, in main
    fm.initialize()
  File "/usr/lib/python2.7/dist-packages/ranger/core/fm.py", line 84, in initialize
    enumerate(self.start_paths))
  File "/usr/lib/python2.7/dist-packages/ranger/core/fm.py", line 83, in <genexpr>
    self.tabs = dict((n+1, Tab(path)) for n, path in
  File "/usr/lib/python2.7/dist-packages/ranger/core/tab.py", line 19, in __init__
    self.path = abspath(expanduser(path))
  File "/usr/lib/python2.7/posixpath.py", line 364, in abspath
    cwd = os.getcwd()
OSError: [Errno 2] No such file or directory
```

**Comment by toonn** [comment]
I'm sorry for the dataloss. Did you quit ranger while it was copying the files?

The error sounds like you're trying to start ranger in a directory that no longer exists.

You're running a very old version of ranger btw, even debian has had 1.7.2 for a long time and we're on 1.9.3 so there's really nothing we can do about this unless you can reproduce it with the current version.

**Comment by alexaldev** [comment]
No, I did not quit ranger but I noticed some strange behavior when it was pasting the currently missing directory. The time it needed for the missing directory was quit abnormal - very small - compared to the previous directories, so that triggered me and expected the above result.

I will try with the new version(I need a whole system upgrade btw) and let you know. 

Thanks for your reply, I believe we can close the issue.
